src/app/public/modules/auth-http/auth-token.ts

```diff
@@ -0,0 +1,78 @@
+export interface SkyAuthToken {
```

Comment:
Just want to make sure I understand the behavior correctly.  If the JWT suddenly contains a new property that's not represented here, that wouldn't cause any JS errors, right?  We could release an update with that property, but it wouldn't be a requirement unless the consumer wanted to access that property first-class.

Reply:
If you wanted to allow any other properties on the JWT without causing lint errors, might be good to add the following to the end of the interface:

```
export interface SkyAuthToken {
  // ...

  [key: string]: any;
}
```

Reply:
(But I guess these linting errors would only show up if a person is attempting to set or get a property from the returned token that isn't specifically typed.)

Reply:
Type checks are only done at build time; because there's no "deserialization" process where there's a class definition that exists at runtime and a deserializer that sets properties on an instance of that object, there would be no runtime impact if a new property were added to the JWT.  If a new property were added, we could simply add it to the interface for the benefit of our consumers.  Worst case is the consumer needs to access that property before we can publish the updated interface and they cast it as `any` (or cast it to their own interface that contains the property) and then reference the property.